packages/phone-number-privacy/signer/test/end-to-end/get-blinded-sig.test.ts

```diff
@@ -128,25 +119,14 @@ describe('Running against a deployed service', () => {
 
     it('Returns sig when querying succeeds with used request', async () => {
       await replenishQuota(ACCOUNT_ADDRESS2, contractkit)
-      const response = await postToSignMessage(BLINDED_PHONE_NUMBER, ACCOUNT_ADDRESS2, timestamp)
+      const response = await postToSignMessage(BLINDED_PHONE_NUMBER, ACCOUNT_ADDRESS2)
       expect(response.status).toBe(200)
     })
 
     it('Returns count when querying with used request does not increment query count', async () => {
       const queryCount = await getQueryCount(ACCOUNT_ADDRESS2, IDENTIFIER)
       expect(queryCount).toEqual(initialQueryCount + 1)
     })
-
-    it('Returns sig when querying succeeds with missing timestamp', async () => {
```

Comment:
Similarly here we can just check that this succeeds with missing timestamp. We can remove the test later after we're sure that all Valora versions have removed the timestamp field.